src/pages/[platform]/build-a-backend/data/customize-authz/public-data-access/index.mdx

```diff
@@ -81,16 +81,16 @@ const schema = a.schema({
 In your application, you can perform CRUD operations against the model using `client.models.<model-name>` with the `iam` auth mode.
 
 <Callout info>
-If you're not using the auto-generated **amplifyconfiguration.json** file, then you must set the Amplify Library resource configuration's `allowGuestAccess` flag to `true`. This lets the Amplify Library use the unauthenticated role from your Cognito identity pool when your user isn't logged in.
+If you're not using the auto-generated **amplify_outputs.json** file, then you must set the Amplify Library resource configuration's `allowGuestAccess` flag to `true`. This lets the Amplify Library use the unauthenticated role from your Cognito identity pool when your user isn't logged in.
 
 <Accordion title="Amplify configuration">
 ```ts title="src/App.tsx"
 import { Amplify } from "aws-amplify";
-import config from "../amplifyconfiguration.json";
+import outputs from "../amplify_outputs.json";
 
 Amplify.configure(
   {
-    ...config,
+    ...outputs,
```

Comment:
I don't think this spread operation is needed anymore for guest access to be enabled